File: Example/athmovil-checkout-objc/Edit/Cells/CheckoutEditFooterTableViewCell.m
//
//  CheckoutEditFooterTableViewCell.m
//  athm-checkout-demo-objc
//
//  Created by Cristopher Bautista on 7/10/18.
//  Copyright © 2018 EVERTEC. All rights reserved.
//

#import "CheckoutEditFooterTableViewCell.h"

@implementation CheckoutEditFooterTableViewCell
@synthesize descLabel;

+(NSString*)reuseIdentifier {
    return @"CheckoutEditFooterTableViewCell";
}

- (void)awakeFromNib {
    [super awakeFromNib];
    // Initialization code
    [self setDescLabelStyle];
}

-(void) setDescLabelStyle {
    descLabel.textAlignment = NSTextAlignmentLeft;
    descLabel.numberOfLines = 0;
    descLabel.textColor = [UIColor grayColor];
    descLabel.font = [UIFont systemFontOfSize:12];
    descLabel.text = NSLocalizedString(@"checkout_edit_footer_description", comment: "");
}

- (void)setSelected:(BOOL)selected animated:(BOOL)animated {
    [super setSelected:selected animated:animated];

    // Configure the view for the selected state
}

@end

File: checkout-demo-swift/checkout-demo-swift/Checkout/Edit/CheckoutEditFooterTableViewCell.swift
//
//  CheckoutEditFooterTableViewCell.swift
//  athm-checkout-demo
//
//  Created by Leonardo Maldonado on 5/5/18.
//  Copyright © 2018 Leonardo Maldonado. All rights reserved.
//

import UIKit

class CheckoutEditFooterTableViewCell: UITableViewCell {
    
    static let reuseIdentifier: String = "CheckoutEditFooterTableViewCell"

    @IBOutlet weak var descLabel: UILabel!
    
    override func awakeFromNib() {
        super.awakeFromNib()
        
        setDescLabelStyle()
    }
    
    fileprivate func setDescLabelStyle() {
        descLabel.textAlignment = .left
        descLabel.numberOfLines = 0
        descLabel.textColor = .gray
        descLabel.font = UIFont.systemFont(ofSize: 12)
        descLabel.text = "Payment will be sent to the account of the provided\n" +
        "Public Token for the provided Payment Amount."
    }
}
